src/foam/u2/md/OverlayDropdown.js

```diff
@@ -180,6 +180,7 @@ foam.CLASS({
     function onMouseLeave(e) {
       console.assert(e.target === this.dropdownE_.el_(),
           'mouseleave should only fire on this, not on children');
+      if ( e.toElement?.nodeName == 'DROPDOWN' ) return;
```

Comment:
Could you also say: if ( e == this.dropdownE_ ), which would be clearer?

Reply:
Not really, this is to prevent a parent dropdown from being hidden when moving to a child, I'll add comments to clarify